### PR TITLE
[Refactor] migrate all assets to webpacker (JS CSS and images)

### PR DESCRIPTION
**Pull request by jjf21**
- [x] https://github.com/thoughtbot/administrate/wiki/Rails-6-&-Webpacker > Follow this
- [x] move stylesheets
- [x] move images
- [x] compare new bundle size to old one (cf below)

change list: 
- all CSS, JS and images moved from `app/assets` to `app/webpacker`
- `stylesheet_link_tag` becomes `stylesheet_pack_tag`
- `image_tag` becomes `image_pack_tag`
- for SCSS packs to be created, they NEED to have a JS pack associated to it.
- FontAwesome upgraded from 4 to 5:
  - some icons changed names
  - old icons with `-o` suffix like `.fa.envelope-o` become `.far.envelope`
- `print` stylesheet was included independently in the `app/views/common/_meta` partial, I've moved it to be imported and bundled together with the main `application` stylesheet. it's tiny
- explode `require 'rails/all'` into separate requires in order to exclude sprockets (even though it still seems to be loading

upon releasing : 
- [ ] extensive test of the app on demo:
  - [ ] fonts
  - [ ] all layouts: 
    - [ ] administrate
    - [ ] landing
    - [ ] application
    - [ ] application_users
    - [ ] mail
  - [ ] images, images inserted through CSS, favicon etc
  - [ ] FA icons 
  - [ ] js interactions
    - [ ] timepickers
  - [ ] recheck lighthouse score (current on demo: 73)

comparison of bundle sizes before and after seems constant (but way too high) for application and landing packs (done using `SECRET_KEY_BASE=test RAILS_ENV=production rails assets:precompile` on both master and this branch):

Screenshot 2020-04-16 at 17 28 14

### CODE REVIEW

**Comment by adipasquale** [comment]
niiice @jjf21 and did you try moving CSS to webpacker too ? 

> That's it. If you want to load the stylesheet through webpacker as well, create a new css pack, and move over the entire administrate directory. Same steps as above but then for the stylesheet 💯

if we want to remove sprockets as well we'll have to

**Comment by jjf21** [comment]
Work still in progress but we are trying to move all assets yes!

**Comment by adipasquale** [comment]
@jjf21 do you want me to have a look and try to move administrate's CSS assets to webpack so we can finish this PR?

**Comment by jjf21** [comment]
Using review app could be useful ?

**Comment by adipasquale** [comment]
yes a review app would be nice, but I'm not sure it's strictly necessary nor worth the effort.. I don't know how hard it would be for rdv-solidarites, but I've always found it quite difficult to setup review apps. 

for this particular PR, I believe if one of us does another round of checks locally and that we're then 2 to check things simultaneously on the demo environment it should be okay

File: app/webpacker/stylesheets/structure/_left-menu.scss [hunk 1]
```diff
@@ -1,9 +1,9 @@
-// 
+//
 // menu.scss
 //
 
 body.enlarged .logo{
-  content:url(asset_path("logos/logo_small.svg"));
```
Comment (adipasquale) [inline app/webpacker/stylesheets/structure/_left-menu.scss:6]: I don't understand what we're doing with that line, but it seems to still work once migrated.

here is how it looks right now

Screenshot 2020-04-16 at 16 25 39

Comment (jjf21) [reply app/webpacker/stylesheets/structure/_left-menu.scss:6]: Capture d’écran 2020-04-16 à 16 36 04
I don't have the same logo on demo ?

Comment (adipasquale) [reply app/webpacker/stylesheets/structure/_left-menu.scss:6]: oh right thanks for checking. you're right it doesn't behave the same on chrome and FF. and there are a lot of different width breakpoints

**Pull request merged**